src/main/kotlin/org/bundleproject/utils/Constants.kt

```diff
@@ -0,0 +1,19 @@
+package org.bundleproject.utils
+
+import io.ktor.client.*
+import io.ktor.client.engine.apache.*
+import io.ktor.client.features.json.*
+import io.ktor.http.*
+
+val port = System.getenv("PORT")?.toIntOrNull() ?: 8080
+
+val httpClient = HttpClient(Apache) {
```

Comment:
Could this be moved into another file (could be named HttpClient), it's not really a constant :P